pygazebo/pygazebo.cc

```diff
@@ -441,13 +442,11 @@ class World {
 };
 
 void Initialize(const std::vector<std::string>& args, int port=0, bool quiet=false) {
-  static std::once_flag flag;
   if (port != 0) {
     std::string uri = "localhost:" + std::to_string(port);
     setenv("GAZEBO_MASTER_URI", uri.c_str(), 1);
   }
 
-  std::call_once(flag, [&args, quiet]() {
```

Comment:
If shutdown the environment by gazebo::shutdown and load a new one in the same thread (like the test of grocery gournd does), this call once flag prevent the initialization. So it is removed. 
I can't find the possibility of double call initialize(). From the test everything seems fine. But I'm not sure the consequence of removal of runonce. If I missed something  and this is necessary please tell me.

Reply:
call_once is to prevent creating multiple server if Initialize is called multiple times. Since you want to do is to allow Initialize() after shutdown, we can explicit include this logic:
```c++
g_initialized  =false
void Close() {
   g_initialized = false;
...
}
void Initialize(...) {
   if (!g_initialized) {
     start server
     ...
     g_initialized = true
    }
}
```

Reply:
Done